Integrations/integration-Kenna.yml

```diff
@@ -0,0 +1,487 @@
+description: Kenna is a Risk Intelligence & Vulnerability platform that enables InfoSec
+  teams to prioritize and remediate vulns faster.
+configuration:
+- display: Server URL for the search API (e.g. https://192.168.0.1)
+  name: url
+  defaultvalue: https://api.kennasecurity.com
+  type: 0
+  required: false
+- display: Kenna API key
+  name: key
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: "false"
+  type: 8
+  required: false
+- display: Do not validate server certificate (insecure)
+  name: insecure
+  defaultvalue: "true"
+  type: 8
+  required: false
+script:
+  script: |
+    var apiKey = params.key;
+    var url = params.url;
+
+    function searchVulnerabilities(args) {
+        var urlQuery = {};
+            argKeys = Object.keys(args);
+            for (var i = 0; i < argKeys.length; i++) {
```

Comment:
instead of doing loop, do this more explicitly
```
var argKeys = {
    'min_risk_meter_score': 'min-score',
    'top_priority': 'top-priority',
    .....etc
}
```

Reply:
These are not mandatory, so if user does not insert them, in the way you suggested, they will be `undefined` which will cause errors.